README.md

```diff
@@ -47,11 +47,24 @@ If you have any questions or want to dig into other works we considered, feel fr
 
 ### Civics
 
-Topic description coming soon!
+A major lense through which governance of communities is understood is civics and citizenship. Even our own texts talk about "a civic layer for the web." But it's not clear that we really know what we mean by that! In order to think about communities, and more compellingly, what forms of space and collective action could be built around decentralized forms of governance, let's read about civics!
 
 **Readings:**
 
-Coming soon! Readings should appear at least 1 month before the date of discussion.
+1. Iseult Honohan, Chapter V "Common goods and public virtue" in *[Civic Republicanism](https://www.taylorfrancis.com/books/e/9781134616114/chapters/10.4324/9780203460894-10)*. 
+    This is a somewhat academic text, and to some extent refers to concepts that are defined in the first (mostly historical) part of the book. Still, it's really very interesting and helpfully discusses the tension between a liberal and "republican" view of society and the rights and duties it confers. If it feels long and difficult, maybe concentrate on the sections labelled "Introduction", "The Common Good", "Is Civic Virture Oppressive?", "Specifying Civic Virtue", and "Conclusion" -- together these are about half the full chapter, and should (we hope!) be intelligible without the remaining sections. 
+
+3. Paul Frazee [Information Civics](https://infocivics.com/). Available at: [https://infocivics.com/](https://infocivics.com/). THis is also a somewhat philosophical text, but focused on the new possibiliies that information technology opens up for civic engagement. Hopefully we can bring (1) and (3) (if you read that optional reading) into conversation with this text. 
+4. *Optional* Johnson, P., & Robinson, P. (2014). [Civic hackathons: Innovation, procurement, or civic engagement?](https://onlinelibrary-wiley-com.myaccess.library.utoronto.ca/doi/full/10.1111/ropr.12074) Review of Policy Research, 31(4), 349-357. Available at: [https://onlinelibrary.wiley.com/doi/full/10.1111/ropr.12074](https://onlinelibrary-wiley-com.myaccess.library.utoronto.ca/doi/full/10.1111/ropr.12074) (open access) -- Read the Intro and Conclusion, Skim the rest . This text is largely a reminder of where many of us come from, and a prompt to htink about what we might want to be doing differnetly or better. 
```

Comment:
Typos:
- htink
- differnetly

Reply:
The URL as written here (https://onlinelibrary.wiley.com/doi/full/10.1111/ropr.12074) is different from where the link points (UT Library: https://onlinelibrary-wiley-com.myaccess.library.utoronto.ca/doi/full/10.1111/ropr.12074). Should we link them each separately?

Reply:
People with SF library cards (probably just me?) can get to it through the article database search here: https://sfpl.org/index.php?pg=2000028601 (I couldn’t figure out how to make a permalink to the article record that worked, even though I *could* link to the actual PDF, but that would almost certainly violate some terms. 😠)